queenbee/schema/workflow.py

```diff
@@ -200,31 +294,6 @@ def hydrate_templates(workflow, wf_value=None):
     return wf_value
 
 
-def list_artifacts(obj):
```

Comment:
Why did we remove this?

Reply:
I implemented it for each object. For instance `workflow.artifacts` returns the artifacts in that workflow and so on.

Reply:
Aaaah I see it now. Neat!